internal/plugin/plugin.go

```diff
@@ -342,9 +344,38 @@ func calculateGitModified(ctx context.Context, pluginYamlPath string) (bool, err
 	return strings.TrimSpace(output.String()) != "", nil
 }
 
-// changedFiles contains data from the tj-actions/changed-files action.
-// See https://github.com/tj-actions/changed-files#outputs for more details.
-type changedFiles struct {
-	AnyModified      string   `env:"ANY_MODIFIED"`
-	AllModifiedFiles []string `env:"ALL_MODIFIED_FILES"`
+func getBaseRefFromEnv(ctx context.Context, lookuper envconfig.Lookuper) (string, error) {
+	type env struct {
+		BaseRef string `env:"BASE_REF"`
+	}
+	var e env
+	if err := envconfig.ProcessWith(ctx, &envconfig.Config{
```

Comment:
We can probably just switch to reading the env var here without envconfig.

Reply:
Didn't want to change too much how it was, but sure I can do that